Core/Base/DataBase.php

```diff
@@ -296,17 +296,38 @@ public function getEngine()
      *
      * @return array
      */
-    public function getIndexes(string $tableName): array
+    public function getAllIndexes(string $tableName): array
```

Comment:
Qué diferencia hay entre esta función y la otra?

Reply:
getIndexes devuelve solo los indices gestionados por facturascripts(prefijo 'fs_') y getAllIndexes devuelve todos los indices de la tabla sin distinción.

de esta forma no se tocan indices preexistentes o creados fuera de facturascripts y se identifican mejor los indices pertenecientes a facturascripts.




 /**
     * Returns an array with the FacturaScripts indexes of a given table.
     *
     * @param string $tableName
     *
     * @return array
     */
    public function getIndexes(string $tableName): array
    
    
    /**
     * Returns an array with the indices of a given table.
     *
     * @param string $tableName
     *
     * @return array
     */
    public function getAllIndexes(string $tableName): array